drivers/sensors/pmw3360.h

```diff
@@ -56,17 +56,6 @@
 #    error "No chip select pin defined -- missing PMW3360_CS_PIN"
 #endif
 
-/*
-The pmw33660 and pmw3389 use the same registers and timing and such.
-The only differences between the two is the firmware used, and the
-range for the DPI. So add a semi-secret hack to allow use of the
-pmw3389's firmware blob.  Also, can set the max cpi range too.
-This should work for the 3390 and 3391 too, in theory.
-*/
-#ifndef PMW3360_FIRMWARE_H
-#    define PMW3360_FIRMWARE_H "pmw3360_firmware.h"
-#endif
```

Comment:
Removing this requires pmw3360.c be updated.
https://github.com/qmk/qmk_firmware/blob/cfcbc089b77295bda68aba39ab0e516cf1f78a1e/drivers/sensors/pmw3360.c#L24
should now be
```c
#include "pmw3360_firmware.h"
```

Reply:
Ah yeah I missed that thanks.
I guess that part didn't technically need to be removed, but with this correction it does makes all the drivers that need to load firmware consistent.